openslides_backend/action/actions/motion/mixins.py

```diff
@@ -31,3 +33,15 @@ def is_submitter(self, submitter_ids: List[int], state_id: int) -> bool:
             s.get("meeting_user_id") in (user.get("meeting_user_ids") or [])
             for s in submitters
         )
+
+
+class AmendmentParagraphHelper:
+    def validate_amendment_paragraph(self, instance: Dict[str, Any]) -> None:
+        if not isinstance(instance["amendment_paragraph"], dict):
+            raise ActionException("Wrong amendment_paragraph except dict")
+        for key, html in instance["amendment_paragraph"].items():
+            if not str.isdigit(key):
+                raise ActionException(f"amendment_paragraph {key} not allowed.")
```

Comment:
I would rather put this directly in the schema - you cannot use the `optional_properties` then, but have to enter the schema manually into the `additional_optional_properties`, but it's cleaner and more uniform than checking the schema here in the code.

Reply:
I moved both check into the schema.